doc/nrf/ug_nrf9160.rst

```diff
@@ -24,33 +24,94 @@ For more details on the SiP, see the `nRF9160 product website`_.
 
-To use any of the nRF9160 samples, you must first compile and flash the
-**Secure Partition Manager** sample.
+All nRF9160 samples require the :ref:`secure_partition_manager` sample.
 It provides a reference implementation of a Secure Partition Manager firmware.
 This firmware is required to set up the nRF9160 DK so that it can run user applications in the non-secure domain.
 
-Note that the **Secure Partition Manager** sample is automatically included in the build for the nrf9160_pca10090ns board.
-To disable the automatic inclusion of the **Secure Partition Manager** sample, set the option ``CONFIG_SPM=n`` in the project configuration.
+The Secure Partition Manager sample is automatically included in the build for the ``nrf9160_pca10090ns`` board.
+To disable the automatic inclusion of the Secure Partition Manager sample, set the option :option:`CONFIG_SPM` to "n" in the project configuration.
+
+
+Application
+-----------
+
+The user application runs in the non-secure domain.
+Therefore, it must be built for the ``nrf9160_pca10090ns`` board.
+
+The application image might require other images to be present.
+Depending on the configuration, all these images can be built at the same time in a :ref:`multi-image build <ug_multi_image>`.
+
+All nRF9160 samples include the :ref:`secure_partition_manager` sample, which can be enabled or disabled with the :option:`CONFIG_SPM` option.
+Some also include the :ref:`bootloader` sample (:option:`CONFIG_SECURE_BOOT`) and :doc:`mcuboot:index` (:option:`CONFIG_BOOTLOADER_MCUBOOT`).
+
+BSD socket
+----------
+
+.. TODO: do we have a bit more information? At least some link?
+
+The **BSD Socket** library must be embedded in the application because
+BSD sockets are the primary interface to the LTE modem.
+
+
+LTE modem
+=========
 
-For detailed documentation of the sample, see :ref:`secure_partition_manager`.
+The LTE modem handles the LTE communication.
+It is controlled through `AT commands <AT Commands Reference Guide>`_.
+
+The firmware for the modem is available as precompiled binary.
+You can download it from the `nRF9160 product website (compatible downloads)`_.
+
+There are two ways to update the modem firmware:
+
+Full upgrade
+  A full upgrade of the modem firmware requires a wired connection.
+  The upgrade is done through the nRF Connect Programmer, which is part of `nRF Connect for Desktop`_.
+  See `Updating the nRF9160 DK cellular modem`_ in the nRF Connect Programmer User Guide for instructions.
+
+Delta patches
+  Delta patches are upgrades that contain only the difference from the last version.
+  A delta patch can only upgrade the modem firmware from one specific version to another version.
+  When applying a delta patch, you must therefore ensure that this patch works with the current firmware version on your device.
+  Delta patches are applied as firmware over-the-air (FOTA) upgrades.
+  See :ref:`nrf9160_ug_fota` for more information.
+
+.. TODO: where can the user get these patches?
```

Comment:
@hakonfam ?

Reply:
https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF9160-DK/Download

Here you find the delta patches. note they are contained in the zip files.